spec/compile-cache-spec.coffee

```diff
@@ -81,6 +82,7 @@ describe 'CompileCache', ->
 
       error = new Error("Oops")
       expect(error.stack).toBe 'a-stack-trace'
+      console.log('stack ' + error.getRawStack())
```

Comment:
Oops, nice catch. This test was randomly failing without useful info so was logging it. Will remove.